wavefront/server/modules/plugins_module/plugins_module/controllers/cloud_storage_controller.py

```
@@ -0,0 +1,102 @@
+                        for item in data
+                    ]
+
+        return JSONResponse(
+            status_code=status.HTTP_200_OK,
+            content=response_formatter.buildSuccessResponse({'data': data}),
+        )
+    except (json.JSONDecodeError, ValueError) as e:
+        return JSONResponse(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            content=response_formatter.buildErrorResponse(
+                f'Failed to parse file as {type.value}: {str(e)}'
```

Ruff (0.15.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)